### PR TITLE
cleanup casks

### PR DESCRIPTION
**Pull request by DominicVonk**

### CODE REVIEW

**Comment by Jason2866** [comment]
Your files in the PR are corrupted

**Review by DominicVonk — COMMENTED** [review]
Clean up

**Review by DominicVonk — COMMENTED** [review]
Clean up

File: Casks/wine-crossover-20.rb [hunk 1]
```diff
@@ -0,0 +1,57 @@
+cask 'wine-crossover-20' do
```
Comment (Gcenx) [inline Casks/wine-crossover-20.rb:1]: I’ve long removed this cask and archive

File: Casks/unofficial-wineskin.rb [hunk 2]
```diff
@@ -1,7 +1,12 @@
-cask "unofficial-wineskin" do
-  version "1.8.4.2"
+cask 'unofficial-wineskin' do
+  version '1.8.4.2'
   sha256 :no_check
```
Comment (Gcenx) [inline Casks/unofficial-wineskin.rb:4]: This section is duplicated over changing from “ to '

**Comment by Gcenx** [comment]
I’m not even sure why this change would be required considering homebrew-core and homebrew-cask uses “

**Pull request closed without merging**